src/layouts/PageLayout/PageLayout.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types'
 import './PageLayout.scss'
 
 export const PageLayout = ({ children }) => (
-  <div className='container text-center'>
+  <div className='text-center'>
```

Comment:
Here I removed `container` className to get rid of `margin: 0 auto`. Its bad for responsive markup demonstrate here.